app/frontend/src/components/NotificationBar/NotificationBar.tsx

```diff
@@ -0,0 +1,62 @@
+// Eternal Imports
+import { combineClasses } from "components/Utility/utils";
+import React, { useState, useRef } from "react";
+import { CSSTransition } from "react-transition-group";
+import { IconButton } from "components/components";
+// Internal Imports
+import { iconX } from "../../assets/images/images";
+import { auto } from "@popperjs/core";
+// Type declaration for props
+interface NotificationBarProps extends React.PropsWithChildren {
+  autoHidden: boolean; // 1. Added new prop to indicate the bar disappearing on its own
+  color: String;
+  text: String;
```

Comment:
Pinging [this](https://github.com/hackforla/CivicTechJobs/pull/356/files#r1130236100) and [this](https://github.com/hackforla/CivicTechJobs/pull/356/files#r1130236718) in case it was missed last time. :)